app/services/resync_service.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class ResyncService < ApplicationService
+  attr_reader :document
+
+  def initialize(document)
+    @document = document
+  end
+
+  def call
+    if document.live_edition.present?
+      # Present to Publishing API & Asset Manager draft stack
+      PreviewService.call(document.live_edition)
+
+      # Publish document to live stack
+      #  (We can't use PublishService directly as it promotes
+      #  `current_edition` to `live_edition`. We just want to
+      #  resync the two with Publishing API, not change their
+      #  states.)
+      GdsApi.publishing_api_v2.publish(
+        document.content_id,
+        nil, # Sending update_type is deprecated (now in payload)
+        locale: document.locale,
+      )
+
+      # Publish assets to live stack
+      PublishAssetService.call(document.live_edition, document.live_edition)
+    end
+
+    # Present to publishing API & Asset Manager draft stack
+    PreviewService.call(document.current_edition)
+    document.reload
```

Comment:
we shouldn't need to do this too by the way - though I'm guessing it's an artefact from the work in progress.